week1/homework/src/server.js

```diff
@@ -10,6 +10,53 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+
+    switch (request.url) {
+      case '/add':
+        response.writeHead(200, {
+          'Content-Type': 'application/json'
+        });
+
+        response.end(JSON.stringify({
```

Comment:
Since we are duplicating all of this code from line 16 to line 22 with some tiny changes each time, perhaps a DRY function will make look a bit more shorter?

And in that case you will find it even more readable almost as:
```javascript
      case '/add':
        // Short code.
        break;
      case '/state':
        // Short code.
        break;
      case '/subtract':
        // Short code.
        break;
      case '/reset':
        // Short code.
        break;
      default:
        // Short code.
    }
```

Reply:
Thank you for your time and feedback.